### PR TITLE
Hotfix

### PR DESCRIPTION
**Pull request by tuankiet2640**
Hotfix

### CODE REVIEW

**Review by MinhThang2709 — COMMENTED** [review]
LGTM

**Review by MinhThang2709 — APPROVED** [review]
LGTM

**Review by kyphuc99 — APPROVED** [review]
LGTM

File: src/main/java/com/codegym/cgzgearservice/service/impl/ProductServiceImpl.java [hunk 1]
```diff
@@ -80,16 +80,16 @@ public Page<ProductDTO> searchProduct(String searchTerm, Pageable pageable) {
...
-            Join<Product, ProductDetail> productDetailJoin = root.join("productDetail", JoinType.LEFT);
-            Join<ProductDetail, Specification> specificationJoin = productDetailJoin.join("specifications", JoinType.LEFT);
-
-            // Predicate for specification values
-            Predicate specValuePredicate = criteriaBuilder.like(
-                    criteriaBuilder.lower(specificationJoin.get("specValue")),
-                    "%" + searchTerm.toLowerCase() + "%"
-            );
-            predicates.add(specValuePredicate);
+//            // Join to access specification values
```
Comment (daonbm) [inline src/main/java/com/codegym/cgzgearservice/service/impl/ProductServiceImpl.java]: Remove comments

**Review by daonbm — CHANGES_REQUESTED** [review]
Remove comments

**Pull request merged**